docs/source/get_started/computers.rst

```diff
@@ -13,7 +13,7 @@ Remote computer requirements
 Requirements for a computer are:
 
 * It must run a Unix-like operating system
-* The default shell must be ``bash``
+* It must have ``bash`` installed (but ``bash`` does not need to be the default shell)
```

Comment:
add comment: Make sure that your remote bash configuration does not load another shell, because then AiiDA will no longer work.